lib/puppet/parser/functions/consul_sorted_json.rb

```diff
@@ -6,11 +6,13 @@ class << self
 
     def sorted_generate(obj)
       case obj
-        when Fixnum, Float, TrueClass, FalseClass, NilClass
-          return obj.to_json
+        when NilClass
+          return "null"
```

Comment:
I agree, but then I would need to return "'null'" (quotes in the string), but we don't want that. `.to_json` returns a string that is then put into the template. It's confusing, we have layers of strings here...